website/docs/docs/build/sources.md

```diff
@@ -191,6 +191,14 @@ The results of this query are used to determine whether the source is fresh or n
 
 <Lightbox src="/img/docs/building-a-dbt-project/snapshot-freshness.png" title="Uh oh! Not everything is as fresh as we'd like!"/>
 
+### Build models based on source freshness
```

Comment:
overall, i might be misunderstanding but don't really get what this section is trying to communicate? is it trying to tell us about how source freshness works? yr about tests? the header mentions its about building models based on source freshness but i dont really understand how a user would do that based on this info?